Pack-Smart/pack-smart-fe
# Feature/landing page

## What's this PR do?
- Sets up router in `App` with a Switch to Error
- Styles out the `Home` component
- Styles out the `Header` component
- Standardizes primary button styling

## Where should the reviewer start?
- App.js
- Header.js
- Home.js
- mixins.scss

## How should this be manually tested?
- Run `npm start` and view the two new components.
- Enter a random router in the browser to make sure you see "Error" rendered.

## What are the relevant tickets?
- closes #5 
- closes #6 
- closes #7 

## Screenshots/GIFs
- <img width="1431" alt="Screen Shot 2021-02-19 at 5 04 37 PM" src="https://user-images.githubusercontent.com/59029768/108574403-8ed6b100-72d4-11eb-9414-5d91938af703.png">

## Did you update the README?
- No
